storage/store/inmemory/unsynchronized/light_transaction_results.go

```diff
@@ -0,0 +1,110 @@
+package unsynchronized
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/storage"
+	"github.com/onflow/flow-go/storage/store"
+)
+
+type LightTransactionResults struct {
+	// TODO: A mutexes aren't strictly necessary here since, by design, data is written only once
+	// before any future reads. However, we're keeping it temporarily during active development
+	// for safety and debugging purposes. It will be removed once the implementation is finalized.
+	store     map[string]flow.LightTransactionResult
+	storeLock sync.RWMutex
```

Comment:
I think you can just use a single mutext to keep it simple since the plan is to remove them in the future anyway